processing/src/main/java/io/druid/query/filter/DruidDoublePredicate.java

```diff
@@ -19,12 +19,32 @@
 
 package io.druid.query.filter;
 
-
+@SuppressWarnings("unused")
 public interface DruidDoublePredicate
```

Comment:
Is this class an `ExtensionPoint` or not? If yes, please add annotation and Javadocs to methods.

Reply:
not an extension point.